### PR TITLE
Vs feature foryou screen

### PR DESCRIPTION
**Pull request by verasunyy**
-add for you screen
-link faved item with heart button in items and item page

### CODE REVIEW

File: client/js/components/ImageCarousel/styles.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     position: 'relative',
   },
+  // itemImageContainer: {
```
Comment (iyuvraajsingh) [inline client/js/components/ImageCarousel/styles.js]: Please remove commented code.

File: client/js/components/ImageCarousel/styles.js [hunk 2]
```diff
@@ -9,6 +9,10 @@ const styles = StyleSheet.create({
...
+  //   height: 230,
+  //   width: '100%',
+  // },
   imageContainer: {
     width: '100%',
     height: 430,
```
Comment (iyuvraajsingh) [inline client/js/components/ImageCarousel/styles.js]: Use %s here if it is feasible.

**Review by iyuvraajsingh — CHANGES_REQUESTED** [review]
Please do the necessary changes.
Thanks

File: client/js/components/ImageCarousel/styles.js (hunk 1 above)
Comment (verasunyy) [reply client/js/components/ImageCarousel/styles.js]: deleted

File: client/js/components/ImageCarousel/styles.js (hunk 2 above)
Comment (verasunyy) [reply client/js/components/ImageCarousel/styles.js]: I can't change this to percentage the whole carousel changed

**Review by iyuvraajsingh — APPROVED** [review]

**Pull request merged**